BB-SC/MWLayer/Power-Management/BB_Power_Management.md

```diff
@@ -0,0 +1,84 @@
+
+# Power Management
+
+## BB Tags(s)
+<!-- Tag(s) define in which area(s) (cloud, in-vehicle) the BB is executed, and what type of BB it is (tool, process, microservice) -->
+BB-SC
+
+## Functional Clusters
+<!-- In which Functional Cluster the BB be located; if none of the existing fit new required -->
+Power Management
+
+## Layer
+<!-- 1, 2a, 2b, 3 -->
+<!-- NOTE: Flag -->
+MWLayer
+
+## Known Implementation
+
+## ID (unique name)
+
+## Description
+<!-- General Description of the BB -->
+Control the handling of vehicle power in various ECU by provides generic
+services for handling power up and power down phases of the vehicles, to
+manage vehicle power transition phase and energy efficiency.
+It handles decision for power control based on electronic dependency and
+power cascading sequence. It is usually a multi-layer approach, where
+states are managed locally for each SW partition, for each sub system and
+for the complete vehicle system It is multi-layer approach
```

Comment:
For Linux systems, systemd and how it can help with this concern is probably a highly interesting topic to investigate for this BB.

Reply:
Agreed. Will be added to description